node/prepublish.js

```diff
@@ -43,15 +43,14 @@ var repoUtils = require('./repoUtils');
 // Use npm.md as the README for the package.
 var absGitRepoPath = path.resolve(path.join(__dirname, '..', '..', '..'));
 var readmePath = path.join(absGitRepoPath, 'README.md');
-var readmeBackupPath = readmePath + '.orig';
+var readmeBackupPath = 'ORIGINAL.md';
```

Comment:
Yes - ```npm pack``` is kinda annoying. I have a local script that runs ```pack``` and then ```postpublish```, so that I don't have to do it twice.